neurom/features/neurite_features.py

```diff
@@ -109,6 +110,19 @@ def number_of_sections_per_neurite(neurites, neurite_type=TreeType.all):
     return (_sec.count(n) for n in neurites if _ttc(neurite_type)(n))
 
 
+@as_neurite_list
+def total_length_per_neurite(neurites, neurite_type=TreeType.all):
+    '''Get an iterable with the total length of a neurite for a given neurite type'''
+    return (sum(_sec.length(ss) for ss in isection(n))
```

Comment:
@juanchopanza I think you underestimate the complexity of the code needed in order to do these one liners.

For example a user must be familiar with the iterator design of NeuroM, the sections module, the type checker etc. I would put a bet that nobody who is not familiar with NeuroM, can write this one liner without spending at least one day to understand the structure of NeuroM.

Thus, I don't see  the harm in having these one liners accessible by users who have more important things to work on that spending their time understanding NeuroM code.

I should have put the total length functions myself when I created the features module, but I somehow forgot and thus asked Leda to do it because she needed this functionality anyway.